.github/workflows/publish_docker_compose.yml

```diff
@@ -0,0 +1,44 @@
+name: Publish Docker Compose
+on:
+  push:
+    branches:
+      - develop
+      - master
+      - 'release/**'
+      - 'feature/**'
```

Comment:
Do we need one for feature/ as well? Is there an artifact in dockerhub that I can run a quick test?